test/integration/controller/jobset_controller_test.go

```diff
@@ -1257,6 +1258,66 @@ var _ = ginkgo.Describe("JobSet controller", func() {
 				},
 			},
 		}),
+		ginkgo.Entry("jobset with VolumeClaimPolicies should create PVCs", &testCase{
```

Comment:
Can we add a test for retention polciy Retain but include a delete and recreate of a JobSet with update volume claims?